docs/leios-design/README.md

```diff
@@ -702,6 +704,19 @@ The BLS voting mechanism relies on a pairing-based signature scheme defined over
 This approach is advantageous because it enables the aggregation of public keys and signatures, allowing large groups of participants to collectively signal approval through a single compact artifact. 
 Beyond Leios, the BLS mechanism is also relevant to other Cardano subsystems; Mithril already employs BLS-based aggregation, and Peras is expected to adopt a similar approach in future implementations.
 
+#### Choice of BLS Variant
+
+BLS12-381 signatures can be instantiated in two variants that differ only in which group is used for public keys and which is used for signatures. Both variants are equivalent in security and share the same API surface; they differ only in the size of the encoded artifacts:
+- **MinPk variant**: public keys are 48 bytes, signatures are 96 bytes.
+- **MinSig variant**: signatures are 48 bytes, public keys are 96 bytes.
+This creates a straightforward trade-off: either public keys are smaller (MinPk) or signatures are smaller (MinSig).
```

Comment:
Perhaps add a note that the impact of having a bigger or smaller PoP is not weighing in the decision, because these are only used once at the registration level. But the impact of using min PK on the registration certificates is now clearer.